src/utils/dbmlParser/helper.js

```diff
@@ -133,3 +133,44 @@ exports.getDuplicateEnums = applications => {
         throw new Error(errorMessage);
     }
 };
+
+/**
+ * Validates the incoming DBML script by attempting to parse it.
+ * 
+ * If the DBML parsing is successful, returns an HTTP 200 OK response.
+ * If there is an error during parsing, returns an HTTP 400 Bad Request response
+ * with a detailed error message and the location of the error.
+ *
+ * @param {Object} req - The HTTP request object containing the DBML script in the body.
+ * @param {Object} res - The HTTP response object used to send back the status and error details.
+ * 
+ * @returns {void} Sends an HTTP response based on the result of DBML parsing.
+ */
+exports.validateIncomingDbmlScript = function (req, res) {
+    try {
+        const body = req.body;
+        const dbml = body.dbml;
+        new Parser().parse(dbml, 'dbml');
```

Comment:
Should we create a new parser every time? A constant parser can be defined and reused right?